app/rabbit_helper.py

```diff
@@ -50,10 +50,11 @@ def send_message_to_rabbitmq(message,
     """
     rabbitmq_connection = _create_connection()
     rabbitmq_channel = rabbitmq_connection.channel()
+    hdrs = {'source': 'RECEIPTING', 'channel': 'EQ'}
```

Comment:
maybe call it headers we're not short of characters

Reply:
changed